### PR TITLE
Amlogic settings improvement

### PR DESCRIPTION
**Pull request by arthur-liberman**
Add better description for the SDR to HDR and HEVC workaround configurations.
Change HEVC workaround default state to On.
Add HDR to SDR configuration.

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -8337,25 +8337,35 @@ msgstr ""
...
 #: system/settings/settings.xml
 msgctxt "#14282"
-msgid "SDR2HDR mode"
+msgid "Tone map SDR to HDR"
 msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#14283"
-msgid "Select to change the SDR2HDR mode. Requires reboot."
+msgid "Inteded for HDR enabled displays - not recommended. Turn this On or set to Auto to have HDR10 mode always on. GUI and SDR content will be tone mapped. Requires reboot."
```
Comment (adamg88) [inline addons/resource.language.en_gb/resources/strings.po]: Intended

File: addons/resource.language.en_gb/resources/strings.po [hunk 2]
```diff
@@ -8337,25 +8337,35 @@ msgstr ""
...
 
-#empty strings from id 14282 to 14300
+#: system/settings/settings.xml
+msgctxt "#14284"
+msgid "Tone map HDR to SDR"
+msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#14285"
+msgid "Inteded for SDR displays. Turn this On or set to Auto to have HDR content tone mapped to SDR. This improves color saturation and contrast of HDR content on SDR displays. Requires reboot."
```
Comment (adamg88) [inline addons/resource.language.en_gb/resources/strings.po]: As above

**Comment by arthur-liberman** [comment]
I want to think that the SDR to HDR option will take priority on a HDR display if it's on/auto and the HDR to SDR option will take priority on a SDR display if it's on/auto.
Let's wait for some more feedback on this.

**Comment by adamg88** [comment]
SDR2HDR is silly as you said though so I'm +1 for dropping it in favour of retaining the HDR2SDR option you added however if you want to retain both options we could add a dependency to lock HDR2SDR if SDR2HDR is enabled.

```
          <dependencies>
            <dependency type="enable">
                <condition setting="coreelec.amlogic.sdr2hdr" operator="is">0</condition>
            </dependency>
          </dependencies>
```

**Comment by PelicanHu** [comment]
IMO you can enable both because they works in different scenario (tv is HDR or SDR) they never works together. 
You enable both means if you moves the box from to another tv it will behave as the tv type (sdr/dhr)

**Comment by adamg88** [comment]
SDR2HDR tone maps SDR to HDR and forces everything to HDR

HDR2SDR does the complete opposite

Unless I'm wrong... if not then it would make sense to lock 1 option whilst the other is in use.

**Comment by PelicanHu** [comment]
You can test it and you can see it, just set enable or auto both and see what happens. My guess is on sdr tv it will send sdr signal and  on hdr tv it will send hdr signal all the time independently from the source.

**Comment by Raybuntu** [comment]
SDR2HDR needs to be off by default. Which is fixed by the kernel commit I did. It is annoying to have everything SDR related including GUI output and mapped to HDR.

HDR2SDR on the other hand is for HDR content being played on a TV that can't do HDR. Without this option colors are washed out and gray. Thats why this option needs to be "auto" which means it will check if the TV supports HDR if not it will map it to SDR. It is working pretty good although it is not perfect yet still better than no mapping.

**Comment by PelicanHu** [comment]
I agree with you Ray for the default settings just wanted to tell there is no need any (reverse) logic to disable/enable each other as Adam suggested.

**Comment by arthur-liberman** [comment]
OK, I think we should keep both options for now. I know there are *some* people who do like to have HDR on for everything they watch, for whatever reason. Doesn't cost us much to add this.

**Comment by Raybuntu** [comment]
thanks

**Pull request merged**